library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -864,26 +851,9 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
-    @Test
-    fun getRecommendationResultsWithLabelsAgainstRealResponse() {
```

Comment:
Removed this test because  our recommendation in the test index don't return results so no sponsored products.